### DISCUSSION TITLE
RDKIT MolFromSmiles

### DISCUSSION THREAD

**Discussion by VANIAQFB** [rdkit/rdkit#5617] [Q&A]
Hi everyone!
I am desperate, because when I use **Chem.MolFromSmiles** for the isomeric smile of catechin, RDKIT did't give me the right molecule. 
I try to put two versions of isomeric smile of catechind, and both give me a wrong molecule.

So, what can I do?

Please help me, is for my thesis project.

Thanks 

VANIA

**Comment by gedeck** [comment]
Post your SMILES here; otherwise, nobody can help you with this.

You could also try the SMILES from the Wikipedia page for Catechin (https://en.wikipedia.org/wiki/Catechin; look in the table on the top right).
```
Oc1ccc(cc1O)[C@H]3Oc2cc(O)cc(O)c2C[C@@H]3O
```

**Reply by greglandrum** [reply]
@VANIAQFB, as @gedeck said: unless you share the inputs you used and why you think the outputs are wrong we won't be able to help

**Reply by VANIAQFB** [reply]
Hi! Thanks for your help

Well, I share my input:
```
catechin = Chem.MolFromSmiles('Oc1ccc(cc1O)[C@H]3Oc2cc(O)cc(O)c2C[C@@H]3O') 
catechin
```
---And the output is:

![image](https://user-images.githubusercontent.com/113950976/193893371-e87d1d2f-5f60-4a4b-b535-dec655b5b842.png)


I try with two versions of SMILE for catequin (Wikipedia and PubChem) and give me always the same result. 

VANIA

**Reply by VANIAQFB** [reply]
the right molecule of catechin is 
![image](https://user-images.githubusercontent.com/113950976/193896208-be99e33b-bc18-47f4-83c3-9752d695d004.png)

I check both SMILES of catechin (PubChem and Wikipedia) in a molecule from smiles generator online and the SMILES are right, but the molecule in RDKIT is wrong.

**Reply by greglandrum** [reply]
You haven't explained why you think the RDKit result is wrong. It looks correct to me: the wedges are not drawn to the same bonds as in your drawing, but the absolute stereochemistry of the atoms is, I believe, the same.

**Reply by VANIAQFB** [reply]
@greglandrum 
Hi!

Oh! Yes, you are right! I didn't see this before, both images are equivalents.

I thought  RDKIT gave me a wrong result because the wedges are not drawn to the same bonds as the image of catechin. 

Thanks for all! 

VANIA
